src/Compilers/CSharp/Portable/BoundTree/BoundNode.cs

```diff
@@ -149,6 +150,21 @@ public void ResetCompilerGenerated(bool newCompilerGenerated)
             }
         }
 
+        public bool IsSuppressed
```

Comment:
>IsSuppressed [](start = 20, length = 12)

Should `IsSuppressed` be declared on `BoundExpression` rather than `BoundNode`?

Reply:
The reason I defined it here is that the flags (`_attributes`) are private, and I think it makes sense to keep them so.